src/core/thread/mesh_forwarder.cpp

```diff
@@ -454,6 +500,16 @@ otError MeshForwarder::HandleFrameRequest(Mac::TxFrame &aFrame)
         OT_ASSERT(aFrame.GetLength() != 7);
         break;
 
+#if OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE
+    case Message::kTypeMacEmptyData:
+    {
+        Mac::Address macDestAddr;
+        macDestAddr.SetShort(Get<Mle::MleRouter>().GetParent().GetRloc16());
```

Comment:
```suggestion
        Mac::Address macDestAddr;

        macDestAddr.SetShort(Get<Mle::MleRouter>().GetParent().GetRloc16());
```
nit: empty line after variable declaration.